app/src/main/res/layout-land/section_title.xml

```diff
@@ -25,14 +25,10 @@
 
     <TextView
       android:id="@+id/section_title_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
```

Comment:
Why are these removed? They aren't explicitly defined in the style.

Ditto elsewhere.

Reply:
Reverted.